docs/topics/wasm/wasm-troubleshooting.md

```diff
@@ -0,0 +1,63 @@
+[//]: # (title: Troubleshooting)
+
+Kotlin/Wasm relies on new [WebAssembly proposals](https://github.com/WebAssembly/proposals) like garbage collection, 
```

Comment:
Maybe it's better refering to https://webassembly.org/roadmap/?

Reply:
Note there is no "typed function references" proposal because it is implied by the GC proposal and sometimes treated as part of the GC proposal (actually it was 🤯).